index.js

```diff
@@ -1,19 +1,28 @@
 const sharp = require('sharp');
-const CanvasKitInit = require('canvaskit-wasm/bin/full/canvaskit');
-const { WebGLRenderingContext } = require('gl/src/javascript/webgl-rendering-context');
+const CanvasKitInit = require('canvaskit-wasm/bin/full/canvaskit.js');
+
+const webgl = require('webgl-raub');
+const { Document } = require('glfw-raub');
+console.log('start programm');
+Document.setWebgl(webgl);
+
+const doc = new Document();
+global.document = global.window = doc;
+
+// node 18 fetch used by canvaskit-wasm, without this line it tries to download the wasm file using fetch
+global.fetch = undefined;
```

Comment:
This can be important for us, I think we will have the same problem migrating to nodejs 20